codewof/utils/Weekday.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
+
+
+class Weekday(Enum):
```

Comment:
Any particular reason why the week data is stored this way? We often store choice data like this: https://github.com/uccser/dthm4kaiako/blob/develop/dthm4kaiako/resources/models.py#L85-L96 which is similar to the way mentioned in the Django documentation: https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.choices

Reply:
This enum is only used by send_email_reminders.py, not any forms. Specifically it's used in the get_users_to_email function for checking which day of the week it currently is. I thought it would be more readable (e.g. weekday_num == Weekday.MONDAY instead of weekday_num == 0)